src/main/webapp/WEB-INF/tags/date-range.tag

```diff
@@ -0,0 +1,16 @@
+<%@taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@taglib prefix="spring" uri="http://www.springframework.org/tags" %>
+<%@taglib prefix="uv" tagdir="/WEB-INF/tags" %>
+
+<%@attribute name="from" type="java.time.LocalDate" required="true" %>
+<%@attribute name="to" type="java.time.LocalDate" required="true" %>
+<%@attribute name="pattern" type="java.lang.String" required="false" %>
+
+<c:set var="START_DATE">
+    <uv:date date="${from}" pattern="${pattern}" />
+</c:set>
+<c:set var="END_DATE">
+    <uv:date date="${to}" pattern="${pattern}" />
+</c:set>
+
+<spring:message code="date.range" arguments="${START_DATE};${END_DATE}" argumentSeparator=";"/>
```

Comment:
Kann man hier noch einpacken, wenn startDate == endDate ist das dann nicht "Tag bis Tag" da steht?

Reply:
falls wir irgendwann den fall haben, dass wir beim verwenden der ui komponente nicht wissen ob es ein von bis ist oder nicht, dann vielleicht :-) aktuell wissen wir was wir zur hand haben und wie wir es angezeigt haben wollen.

Reply:
d.h. du würdest es auf jeder Seite extra machen und dort checken ob es der selbe Tag ist und wenn ja was anderes ausgeben?

Reply:
Vermutlich ja. Vermutlich soll an Stelle A z. B. "28. Mai 2021, morgens" stehen und an Stelle B "28. Mai 2021". Für ersteres bräuchte es die `dayLength`. Könnte man ja auch optional dazu geben zur `date-range`, so als Gedankenspiel. Und schon wird's imho viel zu komplex in der UI Komponente.

Keine Ahnung, lass uns schauen wenn wir die Fälle haben. Mir fällt gerade nur der Zeitraum Filter bei z. B. Krankmeldungen ein.